File: C/DistanciaEntreDoisPontos.c
#include <stdio.h>
#include <math.h>

int main(void) {
  
  double x1,x2,y1,y2,a,b,d;
  
    scanf ("%lf" "%lf", &x1, &y1);
    scanf ("%lf" "%lf", &x2, &y2);

    a = x2 - x1;
    b = y2 - y1;
    
    d = sqrt( (a * a) + (b * b) );

      printf ("%.4lf\n", d);

  return 0;
}

File: C/DistanciaEntreDoisPontos.java
import java.util.Scanner;

/**
 *
 * @author Gabriel Nunes dos Santos
 */
public class Main {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        
        double x1,x2,y1,y2,a,b,d;
  
        Scanner entrada = new Scanner(System.in);
        
        x1 = entrada.nextDouble();
        y1 = entrada.nextDouble();
        x2 = entrada.nextDouble();
        y2 = entrada.nextDouble();

        a = x2 - x1;
        b = y2 - y1;
    
        d = Math.sqrt( (a * a) + (b * b) );

        System.out.printf("%.4f\n", d);

    }
    
}
